### ISSUE TITLE
Nouvelle version

### ISSUE THREAD

**Issue by Sicoval31** [ddurieux/glpi_escalation#2]
Bonjour,
ce plugin ne semble plus maintenu depuis 2 ans ... peut-être depuis la sortie du plugin escalade qui malheureusement ne rempli pas le même office.
Existe-t-il un risque a utiliser ce plug-in avec la V9.3 de GLPI (apres modif du fichier setup.php) ?
Envisagez-vous de le rendre compatible avec les dernières versions ?
Peut-être qu'Escalade reprendra les bonnes idées de ce plug-in ?!

Merci à vous et bonne journée

**Comment by Sicoval31** [comment]
Sortie de la nouvelle mouture \0/
Hop, telechargement et installation sur la base de test en 9.4.2 et ... ah ben aucune configuration nulle part et la seule trace que je dégotte se trouve dans les  .... profils et dans la gestion de l'entité (mais offre exactement les mêmes options de config) !!!!???? 
Rien dans les groupes ! Ni dans les tickets !

Où puis je définir que tel groupe escalade vers tel groupe ?

Sur les 4 points mentionnés dans la mise à jour du plugin, je n'en vois qu'un : 4.Gestion de la configuration par entité.
Le 2, le plus intéressant est introuvable :( ( Gérer un workflow d'escalade d'un groupe de technicien à un autre (On défini que tel groupe ne peux escalader que vers tel ou tel autre groupe).
Merci pour un petit coup de pouce ?! une micro doc ?! un fichier texte ?

**Comment by wackou72** [comment]
Bonjour @Sicoval31 
Nous avons exactement le même problème, avez-vous réussi à configurer ce plugin ?
@ddurieux existe t'il une documentation ou bien vérifier ce qui ne va pas ?
Merci d'avance et bonne journée ! :)

**Comment by Sicoval31** [comment]
non, j'ai mis en pause les tests sur ce plugin en attendant (en vain ?) un retour. Mais comme David est sur d'autres plugin beaucoup plus sensible et important (fusion), celui-ci passe au 2nd voir 3eme plan !
Patience ...

**Comment by wackou72** [comment]
D'accord, merci pour votre retour :)
Nous allons également patienter.

**Comment by ddurieux** [comment]
c'est bizarre, ca a fonctionné, je vous fais un retour d'ici 2 semaines (je serais en vacances :p)

**Comment by wackou72** [comment]
Super, merci :)

**Comment by wackou72** [comment]
@ddurieux Bonjour,
avez-vous des nouvelles sur ce sujet ?
Merci

**Comment by Sicoval31** [comment]
Bonjour, des nouvelles ?
Une compatibilité prévue avec les V9.5.X de GLPI ?
Bonané

**Comment by Sicoval31** [comment]
Ah, une nouvelle version :)
Compatible V10.XX ?

**Comment by ddurieux** [comment]
9.5 :D
